src/app/api/v1/transfer/[id]/route.ts

```diff
@@ -2,30 +2,8 @@
 
 import { NextRequest, NextResponse } from 'next/server';
 import { validate } from 'uuid';
-import { SplTransfer, TransactionStatus } from '@/app/types/splTransfer';
 import { getSplTransfer } from '@/logic/transactionEngine';
-
-// This type is used to return the transaction to the client.
-// It is used to hide certain fields from the public and to convert the amount to a string.
-export type PublicSplTransfer = Omit<SplTransfer, 'signedTransactionBytes' | 'requestedByIp' | 'amount' | 'estimatedFeeInLamports' | 'feeInLamports' | 'feeInSpl'> & {
```

Comment:
bug fix is to apply this to both routes. didn't realize i had defined these types and functions twice